client/trino-client/src/main/java/io/trino/client/PrestoHeaders.java

```diff
@@ -24,24 +25,26 @@
     public static final String PRESTO_LANGUAGE = "X-Presto-Language";
     public static final String PRESTO_TRACE_TOKEN = "X-Presto-Trace-Token";
     public static final String PRESTO_SESSION = "X-Presto-Session";
+    public static final String PRESTO_ROLE = "X-Presto-Role";
+    public static final String PRESTO_PREPARED_STATEMENT = "X-Presto-Prepared-Statement";
+    public static final String PRESTO_TRANSACTION_ID = "X-Presto-Transaction-Id";
+    public static final String PRESTO_CLIENT_INFO = "X-Presto-Client-Info";
+    public static final String PRESTO_CLIENT_TAGS = "X-Presto-Client-Tags";
+    public static final String PRESTO_CLIENT_CAPABILITIES = "X-Presto-Client-Capabilities";
+    public static final String PRESTO_RESOURCE_ESTIMATE = "X-Presto-Resource-Estimate";
+    public static final String PRESTO_EXTRA_CREDENTIAL = "X-Presto-Extra-Credential";
+
+    // response headers
```

Comment:
it looks like a line comment for next thing.

i find "section-like" comments generally not helpful (they tend to float away), but if you make one, please separate it from the next line or make it otherwise visually more catchy